### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by cmaiw**

### CODE REVIEW

File: client/src/components/FavCheckbox.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  display: flex;
+  width: 16px;
+  height: 16px;
+  border: none;
+  justify-content: center;
+  align-items: center;
+`;
+
+const FavCheckbox = () => {
+  const [checked, setChecked] = React.useState(false);
```
Comment (lmachens) [inline client/src/components/FavCheckbox.js]: You have no benefit of the state right now. Later, you should move `checked` and `onChange` to the component props.

File: client/src/components/EntryInput.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  border-radius: 6px;
+  align-self: center;
+  font-size: 14px;
+  border: none;
+  outline: none;
+  opacity: 0.75;
+`;
+function EntryInput(props) {
+  return (
+    <div {...props}>
```
Comment (lmachens) [inline client/src/components/EntryInput.js]: Why do you have a `div` here? I think you can directly export the styled.input if you change the display to `block`

File: client/src/components/NavButton.js [hunk 3]
```diff
@@ -10,9 +11,16 @@ const NavButton = styled.button`
...
-    outline: ${props => props.theme.colors.quatenary}, opacity:1;
+  &:hover {
+    color: ${props => props.theme.colors.quartenary};
   }
+  &:active {
+  }
+  color: ${props => props.theme.colors.quartenary};
 `;
 
+function NavButton(props) {
```
Comment (lmachens) [inline client/src/components/NavButton.js]: You don't need this additional component. Export your styled component here.
1: remove NavButton
2: rename ThisNavButton to NavButton

File: client/src/icons/CheckmarkIcon.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+const CheckmarkPath = styled.path`
+  fill: ${props => props.theme.colors.background};
+`;
+
+const CheckmarkSvg = styled.svg``;
```
Comment (lmachens) [inline client/src/icons/CheckmarkIcon.js]: No reason to create a styled component here

File: client/src/ressources/feather.svg [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+		<path d="M484.311,27.689c-36.919-36.919-96.775-36.918-133.693,0L306.51,71.797l4,46.663l-46.663-4l-10.466,10.466V237.35
+			l125.211-125.211l21.27,21.27l-43.677,43.678h112.424l15.703-15.703C521.23,124.464,521.23,64.607,484.311,27.689z"/>
+	</g>
+</g>
+<g>
+	<g>
+		<polygon points="79.009,432.991 78.897,411.833 0,490.73 21.27,512 100.167,433.103 		"/>
+	</g>
+</g>
+<g>
```
Comment (lmachens) [inline client/src/ressources/feather.svg]: You can remove these empty `g`

File: client/src/stories/6-EntryInput.stories.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+import EntryInput from '../components/EntryInput';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Entry Input'
+};
+
+export function JournalEntryInput(props) {
+  return (
+    <div onChange={action('key was pressed')} {...props}>
```
Comment (lmachens) [inline client/src/stories/6-EntryInput.stories.js]: This doesn't look correct. Stories won't have props injected.
And the onChange should be on the input and not on the div

File: client/src/stories/7-Form.stories.js [hunk 7]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import EntryInputForm from '../pages/EntryInputForm';
+
+export default {
+  title: 'Entry Input Form'
+};
+
+export function EntryInputFormJournal(props) {
+  return <EntryInputForm {...props}></EntryInputForm>;
```
Comment (lmachens) [inline client/src/stories/7-Form.stories.js]: There are no props in stories.

File: client/.babelrc [hunk 8]
```diff
@@ -0,0 +1,3 @@
+{
+  "plugins": ["emotion"]
```
Comment (lmachens) [inline client/.babelrc]: What is this change? :)

File: client/src/components/EntryInput.js (hunk 2 above)
Comment (cmaiw) [reply client/src/components/EntryInput.js]: thank you...

File: client/src/components/NavButton.js (hunk 3 above)
Comment (cmaiw) [reply client/src/components/NavButton.js]: Works- thank you.

File: client/src/ressources/feather.svg (hunk 5 above)
Comment (cmaiw) [reply client/src/ressources/feather.svg]: I decided to use another svg, so I will remove the file.

File: client/src/stories/6-EntryInput.stories.js (hunk 6 above)
Comment (cmaiw) [reply client/src/stories/6-EntryInput.stories.js]: Maybe here it s not necessary but, for example, the UniversalButton Component will lose hover and active state, when I remove the properties in the story. Is this really intended?

File: client/.babelrc (hunk 8 above)
Comment (cmaiw) [reply client/.babelrc]: I followed a tutorial in styling checkboxes. It was at first not clear that the tutorial used older bibs similar to @emotion/styled. I found another simplified way to style my component- will remove this. thank you.

File: client/src/components/NavButton.js (hunk 3 above)
Comment (cmaiw) [reply client/src/components/NavButton.js]: Thank you...

**Pull request merged**